aat/src/aat/resources/features/F-077/F-077.feature

```diff
@@ -0,0 +1,33 @@
+@F-077
+Feature: CCD Definition Store Api :: GET /api/base-types
+
+  Background:
+    Given an appropriate test context as detailed in the test data source
+
+  @S-329 @Ignore # Response code mismatch, expected: 401, actual: 403 RDM-6628
+  Scenario: must return 401 when request does not provide valid authentication credentials
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [contains an invalid authentication credentials]
+    And it is submitted to call the [fetch all Base Types] operation of [CCD Definition Store]
+    Then a negative response is received
+    And the response [contains 401 unauthorised code]
+    And the response has all the details as expected
+
+  @S-330 @Ignore # Response code mismatch, expected: 403, actual: 200 Is this valid?
+  Scenario: must return 403 when request provides authentic credentials without authorised access to the operation
+    Given a user with [an active profile in CCD, and insufficient privilege to the operation]
+    When a request is prepared with appropriate values
+    And it is submitted to call the [fetch all Base Types] operation of [CCD Definition Store]
+    Then a negative response is received
+    And the response [contains 403 forbidden code]
+    And the response has all the details as expected
+
+  @S-331 #RDM-7131
```

Comment:
RDM-7131 was fully implemented by in another ticket. 7131 is now closed. Could you please remove reference here to it, and reflect any changes required to the test data? See options here, as needed:
https://github.com/hmcts/ccd-api-test-automation-fw/blob/master/src/main/java/uk/gov/hmcts/befta/util/ExpectedValuePlaceholder.java

Reply:
Another thing - happy path scenarios always on top in the feature files.